### ISSUE TITLE
App Becomes Stuck On Pressing Accessibility Button

### ISSUE THREAD

**Issue by NLUG18HV9X7629CA6SP8TN3Y** [standardnotes/app#1247]
On Android 12 when I am in the app and press the accessibility button for color correction (an accessibility button you can turn on in the settings on Android) the app goes to a blank screen and you are unable to do anything further in the app without restarting it.
